packages/flutter_nekoton_bridge/lib/nekoton/core/models/address.dart

```diff
@@ -0,0 +1,32 @@
+import 'package:freezed_annotation/freezed_annotation.dart';
+
+part 'address.freezed.dart';
+part 'address.g.dart';
+
+@freezed
+
+/// Blockchain address
+class Address with _$Address {
+  const Address._();
+
+  const factory Address({
+    required String address,
+  }) = _FromString;
+
+  factory Address.fromJson(Map<String, dynamic> json) =>
+      _$AddressFromJson(json);
+
+  /// Returns address in ellipse form
+  String toEllipseString() => address.length > 6
+      ? '${address.substring(0, 6)}...${address.substring(address.length - 4)}'
+      : address;
+
+  @override
+  String toString() => address;
+
+  /// Compares addresses address
+  int compareTo(Address other) => address.compareTo(other.address);
```

Comment:
Don't you want add `operator ==` and `hashCode` for this and `PublicKey` models, so it will be easier to compare

Reply:
Yeah, sure.

Reply:
*